app/controllers/holds_controller.rb

```diff
@@ -18,6 +19,12 @@ def create
     if @hold.save
       flash[:success] = 'Hold Successfully Created'
       @hold.handle_conflicting_rentals
+
+      # rental that has already started and conflicts with hold, needs to be handled
+      # by a csr
+      if rental = @hold.conflicting_ongoing_rental
+        flash_message :warning, "Ongoing rental with item #{url_for rental}".html_safe
```

Comment:
Let's make the URL here more user friendly. Maybe make "item" clickable with a link to the `url_for`?